tools/pythonpkg/setup.py

```diff
@@ -126,6 +126,7 @@ class build_ext(CompilerLauncherMixin, _build_ext):
     and platform.system() == 'Linux'
     and platform.architecture()[0] == '64bit'
     and platform.machine() == 'x86_64'
+    and "PYODIDE" not in os.environ
```

Comment:
This is the official way to detect if pyodide is used at build time: https://pyodide.org/en/stable/usage/faq.html#how-to-detect-that-code-is-run-with-pyodide